catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/rollover/ByteCountRolloverCondition.java

```diff
@@ -23,26 +24,43 @@
  */
 public class ByteCountRolloverCondition implements RolloverCondition {
 
-    private long byteCountThreshold;
+    private static final long MEGABYTE_TO_BYTE_FACTOR = 1000000;
 
-    public ByteCountRolloverCondition(long byteCountThreshold) {
-        this.byteCountThreshold = byteCountThreshold;
+    public static final long MIN_VALUE = 1;
+
+    public static final long MAX_VALUE = Long.MAX_VALUE / MEGABYTE_TO_BYTE_FACTOR;
+
+    private long megabyteCountThreshold;
```

Comment:
My opinion is that making this a double would introduce some unnecessary rounding complexity.  Is there a use case for specifying 22.2 MB?